src/python/ensembl/io/genomio/database/factory.py

```diff
@@ -87,13 +87,14 @@ def format_db_data(server_url: URL, dbs: List[str], brc_mode: bool = False) -> L
 
 def get_core_dbs_metadata(
     server_url: URL,
+    *,
     prefix: str = "",
-    build: Optional[int] = None,
-    version: Optional[int] = None,
+    build: int | None = None,
+    version: int | None = None,
```

Comment:
Are you opting to follow [PEP604](https://peps.python.org/pep-0604/) ?

Reply:
Yes, this is a change on type hints that came with Python 3.9 if I recall correctly. I find those updates a little bit more readable and sensible than before. _See next comment for more context and opinions on this_